src/components/atoms/IngredientMenu/IngredientMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import { Menu } from "@mantine/core"
+import Image from "next/image"
+
+import { ChevronDown, ChevronUp, Pencil, Trash } from "tabler-icons-react"
+
+interface IngredientMenuProps {
+  showMoveUp: boolean
+  showMoveDown: boolean
+  showEdit: boolean
+}
+
+export const IngredientMenu = ({ showMoveUp = true, showMoveDown = true, showEdit = true }: IngredientMenuProps) => {
+  const commonColor = "#6F6E77"
+  return (
+    <Menu position="bottom-end" width={258}>
+      <Menu.Target>
+        <img src="assets/icons/tabler-icon-dots-vertical.svg" alt="メニュー" width={20} height={20} />
+      </Menu.Target>
+
+      <Menu.Dropdown>
+        {showEdit && (
+          <Menu.Item color={commonColor} icon={<Pencil color={commonColor} />}>
```

Comment:
アイコンのカラーは親から継承するっぽいので不要そうでした！
```suggestion
          <Menu.Item color={commonColor} icon={<Pencil />}>
```

Reply:
消しました！